config/initializers/mongoid.rb

```diff
@@ -8,4 +8,6 @@ def as_json(*args)
       to_s.as_json
     end
   end
-end
\ No newline at end of file
+end
+
+Mongoid::Tasks::Database.create_indexes if Rails.env.production? && !ENV['DISABLE_DB']
```

Comment:
It may not hurt to create indexes here on development and production. I definitely would appreciate the indexes creating automatically during development for me.

Reply:
That would be of use outside of the Docker ecosystem, correct? Seems like a worthwhile enhancement.

Reply:
Yeah that would enhance it for all developers as well. Of note, this change might also affect the Pkgr builds, we currently run this script but it seems it can now be removed: https://github.com/projectcypress/cypress/blob/master/contrib/scripts/after_install.sh

Reply:
The only thing that script does that the Mongoid initializer doesn't is the `db:migrate` task. Does that still need to get run for Pkgr builds?

Reply:
I think it sounds like we came to the conclusion that we don't need the db:migrate at this time, so we should remove it from both, correct?

Reply:
Sounds good to me. I removed it from `after_install`, and didn't add it here.

Reply:
I built a full VM using this pull request and the automatic settings detection worked correctly, so this has not broken any of the production environment related stuff.